tests/threshold/detection_filter-rule-flow/test.yaml

```diff
@@ -0,0 +1,34 @@
+requires:
+  min-version: 8
+
```

Comment:
Cannot you reuse the pcap with `pcap:` instead of duplicating it ?

Reply:
yeah, doing that